homeassistant/components/maxcube.py

```diff
@@ -28,10 +28,12 @@
 NOTIFICATION_TITLE = 'Max!Cube gateway setup'
 
 CONF_GATEWAYS = 'gateways'
+CONF_UPDATE_INTERVAL = 'update_interval'
 
 CONFIG_GATEWAY = vol.Schema({
     vol.Required(CONF_HOST): cv.string,
     vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
+    vol.Optional(CONF_UPDATE_INTERVAL, default=300): cv.positive_int,
```

Comment:
This should be a `cv.time_period` and preferably also `cv.positive_timedelta`.

```python
vol.All(cv.time_period, cv.positive_timedelta)
```

Reply:
I have changed it to cv.time_period, and convert to total seconds (I did not make the original max cube code, and wanted to make as few changes as possible, and the original code has the update interval in seconds - hard coded to 60 seconds)